### ISSUE TITLE
Safari 12 will not load this extension

### ISSUE THREAD

**Issue by csuhta** [livereload/livereload-extensions#52]
<img width="421" alt="c6dsxixxmy" src="https://user-images.githubusercontent.com/769083/45660063-2eb90080-bac5-11e8-84c7-fa1046ab4332.png">

**Comment by superjunky** [comment]
Yes exactly. Does anyone know any alternative browser extension for safari 12?

**Comment by csuhta** [comment]
So I was able to get it to load by following this process, but I don't expect it to work forever. Also you will need to store the unarchived extension contents somewhere, like Dropbox.

https://sixcolors.com/post/2018/09/give-new-life-to-old-extensions-in-safari-12/

**Comment by superjunky** [comment]
When running it with extension builder, this one is helpful aswel:
https://georgegarside.com/blog/macos/install-any-safari-extension-macos-mojave/

**Comment by stychos** [comment]
No more extension builder in Safari 13.

**Comment by tobsen** [comment]
Any news on this?

**Comment by superjunky** [comment]
> Any news on this?

I bet this isn’t going to happen anymore. I started using browsersync https://www.browsersync.io/
